plugins/catalog-graph/src/alpha.tsx

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright 2024 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import {
+  createPageExtension,
+  createPlugin,
+  createSchemaFromZod,
+} from '@backstage/frontend-plugin-api';
+import {
+  compatWrapper,
+  convertLegacyRouteRef,
+} from '@backstage/core-compat-api';
+import { createEntityCardExtension } from '@backstage/plugin-catalog-react/alpha';
+import { catalogGraphRouteRef, catalogEntityRouteRef } from './routes';
+import { ALL_RELATION_PAIRS, Direction } from './components';
+
+type Zod = Parameters<Parameters<typeof createSchemaFromZod>[0]>[0];
+
+function getCompoundEntityRefConfigSchema(z: Zod) {
+  return z.object({
+    kind: z.string(),
+    namespace: z.string(),
+    name: z.string(),
+  });
+}
+
+function getEntityGraphRelationsConfigSchema(z: Zod) {
+  // Mapping EntityRelationsGraphProps to config
+  // TODO: Define how className and render functions will be configured
```

Comment:
I assume that the `className` and the render function are only appropriate for extension overrides, right?

Reply:
Probably yeah

Reply:
Thanks, closing this thread 🙂